context/app/validation_utils.py

```diff
@@ -0,0 +1,17 @@
+import jsonschema
+
+
+def for_each_validation_error(data, schema, callable):
```

Comment:
Maybe add some comments as to what is going on here with respect to why we are appending `_TODO`

Reply:
yes: done.